src/main/java/hu/oe/nik/szfmv/visualization/Gui.java

```diff
@@ -1,32 +1,46 @@
 package hu.oe.nik.szfmv.visualization;
 
+
 import javax.swing.*;
 import java.awt.*;
 
-
+/**
+ * Class for GUI.
+ */
 public class Gui extends JFrame {
-
+    /**
+     * Integer for window width.
+     */
     private final int windowWidth = 1020;
+    /**
+     * Integer for window height.
+     */
```

Comment:
Érthetőbb lett a kód ezekkel a kommentekkel?

Reply:
Azért kerültek be, mert a csodálatos stylecode megköveteli., de olyanoknál is amik véleményem szerint eléggé beszédes névvel rendelkeznek.
Nem tudom, mit kéne egy windowHeight nevű változónál a doc-jába írni. 3 mondaton keresztül részletezni, hogy az ablak szélessége?

Reply:
Beszéljünk erről, ezek szerint a codeStyle beállítása a rossz.

Reply:
@ravaszla Én is erre jutottam. Ezt a Javadoc részt finomítani kell, holnapra nem ígérem.

Reply:
Ha nincs vele más jelentős probléma, merge-ölhetjük codestyle-tól függetlenül, de ahogy elnézem van más baja is.